examples/basic_examples/start_simulation_async.py

```diff
@@ -53,124 +44,145 @@
 from ansys.rocky.core import examples
 
-# Create a temp directory to save the project.
-project_dir = tempfile.mkdtemp(prefix="pyrocky_")
-
-# Launch Rocky and open a project.
-rocky = pyrocky.launch_rocky()
-project = rocky.api.CreateProject()
-project.SaveProject(os.path.join(project_dir, "rocky-testing.rocky"))
-
-###############################################################################
-# Configure the study
-# ~~~~~~~~~~~~~~~~~~~
-study = project.GetStudy()
-
-# Download the STL file that was imported into Rocky to represent a wall.
-file_name = "Lshape_tube.stl"
-file_path = examples.download_file(project_dir, file_name, "pyrocky/geometries")
-wall = study.ImportWall(file_path)[0]
-
-# Create a particle with the default shape (sphere) and size distribution (single
-# distribution with a sieve size of 0.1 m).
-particle = study.CreateParticle()
-
-# Create a circular surface to used as the inlet.
-circular_surface = study.CreateCircularSurface()
-circular_surface.SetMaxRadius(0.8, unit="m")
-
-# Create a rectangular surface to use as the outlet.
-rectangular_surface = study.CreateRectangularSurface()
-rectangular_surface.SetLength(3, unit="m")
-rectangular_surface.SetWidth(3, unit="m")
-rectangular_surface.SetCenter((5, -7.5, 0), unit="m")
-
-# Set the inlet and outlet.
-particle_inlet = study.CreateParticleInlet(circular_surface, particle)
-input_property_list = particle_inlet.GetInputPropertiesList()
-input_property_list[0].SetMassFlowRate(1000)
-outlet = study.CreateOutlet(rectangular_surface)
-
-# Set the motion rotation over the Y axis and apply it on the wall and the
-# rectagular surface used as the outlet.
-motion_frame_source = study.GetMotionFrameSource()
-motion_frame = motion_frame_source.NewFrame()
-motion_frame.AddRotationMotion(angular_velocity=((0.0, 0.5, 0.0), "rad/s"))
-motion_frame.ApplyTo(rectangular_surface)
-motion_frame.ApplyTo(wall)
-
-# The domain settings define the domain limits where the particles are enabled to be
-# computed in the simulation.
-domain = study.GetDomainSettings()
-domain.DisableUseBoundaryLimits()
-domain.SetCoordinateLimitsMaxValues((10, 1, 10), unit="m")
-
-###############################################################################
-# Set up the solver and run the simulation
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-solver = study.GetSolver()
-simulation_duration = 5
-solver.SetSimulationDuration(simulation_duration, unit="s")
-study.StartSimulation(
-    non_blocking=True
-)  # `non_blocking` only available on Rocky 25R1 and onwards.
-
-###############################################################################
-# Postprocess
-# ~~~~~~~~~~~
-# Obtain the in and out mass flows of the particles while the simulation is
-# running.
-particles = study.GetParticles()
-
-import time
+awp_roots = sorted(
+    [k for k in os.environ.keys() if k.startswith("AWP_ROOT")], reverse=True
+)
+last_rocky_version = awp_roots[0]
+if "25.1" in last_rocky_version:
```

Comment:
IMHO the previous version (early exit here) was better. In it we'd just remove the `if` when 25.1 becomes available.

Reply:
The problem is that that exit was breaking the documentation step. IMHO it was way better the other way.

Reply:
Ah, Ok... Let's merge then.